# Lista de Estrutura Sequencial

#### 1. Faça um Programa que mostre a mensagem (print) "Alo mundo" na tela.

In [1]:
print("Olá mundo!")

Olá mundo!


#### 2. Faça um Programa que peça um número (input) e então mostre a mensagem: "O número informado foi [número]."

In [2]:
num = int(input("Digite um numero:"))
print(f"O numero digitado foi: {num}")

Digite um numero:55
O numero digitado foi: 55


#### 3. Faça um Programa que peça dois números e imprima a soma.

In [3]:
num1 = int(input("Digite um numero:"))
num2 = int(input("Digite um numero:"))

print(f"Soma é {num1 + num2}")

Digite um numero:6
Digite um numero:5
Soma é 11


#### 4. Faça um Programa que peça as 4 notas bimestrais de um aluno e mostre a média de todas as notas.

In [4]:
n1 = float(input("Digite nota 1:"))
n2 = float(input("Digite nota 2:"))
n3 = float(input("Digite nota 3:"))
n4 = float(input("Digite nota 4:"))

media = ((n1 + n2 + n3 + n4)/4)

print(f"A media é: {media}")

Digite nota 1:5
Digite nota 2:5
Digite nota 3:5
Digite nota 4:5
A media é: 5.0


#### 5. Faça um Programa que converta metros para centímetros. Você pode pedir o comprimento em metros para o usuário (input).

In [5]:
cm = float(input("Digite cm:"))

m = cm/100

print(m)

Digite cm:56
0.56


#### 7. Faça um Programa que calcule a área de uma sala de um apartamento. Para isso, o seu programa precisa pedir a largura da sala, o comprimento da sala e imprimir a área em m² da sala.

In [6]:
lar = float(input("Digite a largura:"))
com = float(input("Digite o comprimento:"))

print(f"A sala tem {lar * com} metros quadrados!")

Digite a largura:56
Digite o comprimento:56
A sala tem 3136.0 metros quadrados!


#### 8. Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês. Calcule e mostre o total do seu salário no referido mês.

In [7]:
horasMes = float(input("Quantas horas vc trabalha por mes?"))
ganhoHora = float(input("Quanto vc ganha por hora?"))

print(f"Seu salario é: {horasMes * ganhoHora} Reais")

Quantas horas vc trabalha por mes?56
Quanto vc ganha por hora?56
Seu salario é: 3136.0 Reais


#### 9. Vamos criar um conversor de temperatura. Faça um Programa que peça a temperatura em graus Fahrenheit, transforme e mostre a temperatura em graus Celsius.
$C = \frac{5}{9}(F-32)$

In [16]:
f = int(input("Digite a temperatura em farenraiti: "))

c = 5 / 9 * (f - 32)

print(f"{c} Celssius")

Digite a temperatura em farenraiti: 86
30.0 Celssius


#### 10. Faça um Programa que peça a temperatura em graus Celsius, transforme e mostre em graus Fahrenheit.
$F = \frac{9}{5}C + 32$

In [17]:
c = int(input("Digite a temperatura em celsos: "))

f = (c * ( 9 / 5 ) + 32)
print(f"{f} farenraitis")

Digite a temperatura em celsos: 30
86.0 farenraitis


#### 12. Tendo como dados de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal, usando a seguinte fórmula:
$P = 72,7h - 58$

Lembrando que "algoritmo" nada mais é do que um programa, como todos os outros que você vem fazendo

#### 13. Tendo como dado de entrada a altura (h) de uma pessoa, construa um algoritmo que calcule seu peso ideal, utilizando as seguintes fórmulas:
##### a. Para homens: $P = 72,7h - 58$
##### b. Para mulheres: $P = 62,1h - 44,7$

#### 15. Faça um Programa que pergunte quanto você ganha por hora e o número de horas trabalhadas no mês.

#####  Calcule o salário bruto (horas * salario por hora)

##### Calcule o desconto do IR (11% do salário bruto)

##### Calcule o desconto do INSS (8% do salário bruto)

##### Calcule o desconto do sindicato (5% do salário bruto)

##### Calcule o salário líquido (salário bruto - descontos)

#### 16. Faça um programa para uma loja de tintas. O programa deverá pedir o tamanho em metros quadrados da área a ser pintada. Considere que a cobertura da tinta é de 1 litro para cada 3 metros quadrados e que a tinta é vendida em latas de 18 litros, que custam R\$ 80,00. Informe ao usuário a quantidades de latas de tinta a serem compradas e o preço total. (para simplificação nesse momento, não se preocupe em arredondar a quantidade de latas a serem compradas - vamos trabalhar isso em breve)

#### 18. Faça um programa que peça o tamanho de um arquivo para download (em MB) e a velocidade de um link de Internet (em Mbps), calcule e informe o tempo aproximado de download do arquivo usando este link (em minutos).

Detalhe: MB significa megabyte, Mb (com b minúsculo) significa megabit. Um megabit é 1/8 de um megabyte. 